execution/evm/native.go

```diff
@@ -73,13 +74,14 @@ func ecrecoverFunc(state State, caller *acm.Account, input []byte, gas *int64) (
 	recovered, err := secp256k1.RecoverPubkey(hash, sig)
 	if err != nil {
 		return nil, err
+OH NO STOCASTIC CAT CODING!!!!
```

Comment:
🐱

Reply:
Lol you know what that was actually an accidental copypasta from a discussion I was having with @dennismckinnon ... occasionally happens via trackpad... So an accidental code addition referring to a class of accidental code additions.